worker/firewaller/firewaller.go

```diff
@@ -943,7 +942,7 @@ func (fw *Firewaller) ingressRulesForExposedMachineUnit(machine *machineData, un
 }
 
 // TODO(wallyworld) - consider making this configurable.
-const maxAllowedCIDRS = 20
+const maxAllowedCIDRS = 30
```

Comment:
This was changed since we have theoretically dropped a layer to fallback to before defaulting to 0.0.0.0/0

Hence, to compensate, I thought it would be a good idea to increase this variable from an arbitrary value to a slightly higher arbitrary value